config/routes.rb

```diff
@@ -1,16 +1,28 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :topics
-  resources :organizers do
+
+  scope '/admin' do
+    resources :topics, only: [:edit, :new]
+    resources :organizers, only: [:edit, :new]
+    resources :locations, only: [:edit, :new]
+    resources :opportunities, only: [:edit, :new]
+    resources :opportunity_instances, only: [:edit, :new]
+  end
+
+  resources :topics, except: [:edit, :new]
+  resources :locations, except: [:edit, :new]
+  resources :opportunities, except: [:edit, :new]
+
+  resources :organizers, except: [:edit, :new] do
     resources :opportunities
```

Comment:
@justinxreese I was unsure about modifying these nested opportunities resources, because there was already an un-nested version... Not sure if that was on purpose or an oversight. And if I should apply the scope to that nested ``resources :opportunities`` too.